upup/pkg/fi/cloudup/gcetasks/instancetemplate.go

```diff
@@ -247,10 +249,11 @@ func (e *InstanceTemplate) mapToGCE(project string, region string) (*compute.Ins
 
 	if fi.BoolValue(e.Preemptible) {
 		scheduling = &compute.Scheduling{
-			AutomaticRestart:  fi.Bool(false),
-			OnHostMaintenance: "TERMINATE",
-			ProvisioningModel: "STANDARD", // TODO: Support Spot?
-			Preemptible:       true,
+			AutomaticRestart:          fi.Bool(false),
+			OnHostMaintenance:         "TERMINATE",
+			ProvisioningModel:         fi.StringValue(e.GCPProvisioningModel),
+			InstanceTerminationAction: "STOP",
```

Comment:
I am trying to understand why we need stop here. Would you mind providing some context?

Reply:
unfortunately I can not reproduce it now. So, I can remove InstanceTerminationAction here